core/test/functional/api/v2/admin/slugs_spec.js

```diff
@@ -0,0 +1,142 @@
+var should = require('should'),
+    supertest = require('supertest'),
+    testUtils = require('../../../../utils'),
+    localUtils = require('./utils'),
+    config = require('../../../../../server/config'),
+    ghost = testUtils.startGhost,
+    request;
+
+describe('Slug API', function () {
+    let ownerCookie;
+    let ghostServer;
+
+    before(function () {
+        return ghost()
+            .then(function (_ghostServer) {
+                ghostServer = _ghostServer;
+                request = supertest.agent(config.get('url'));
```

Comment:
> .set('Origin', config.get('url'))

Could be globally registered :)

Reply:
Update: We found out it can't be globally registered like this, so keeping it as is for now :)